src/python/dxpy/nextflow/nextflow_builder.py

```diff
@@ -65,7 +65,7 @@ def parse_extra_args(extra_args):
         parser.error(
             "Can't create an applet without specifying a destination project; please use the -d/--destination flag to explicitly specify a project")
 
-    nf_builder_job = dxpy.DXApp(name=get_importer_name()).run(app_input=input_hash, project=build_project_id,
+    nf_builder_job = dxpy.DXApp("app-GP8gZ8j0FKzB35v0yFzgQxb9").run(app_input=input_hash, project=build_project_id,
```

Comment:
What is the reason behind using app id instead of "nextflow_pipeline_importer" ?

Reply:
Good question. I used it for the purpose of testing the file-apps APPS-1611 only and added an explanation to the [description](https://github.com/dnanexus/file-apps/pull/708) and the ticket.. I'll revert the commit so that it is back to `nextflow_pipeline_importer`.